src/org/exist/start/start.config

```diff
@@ -51,7 +51,7 @@ lib/core/log4j-core-%latest%.jar                   always
 lib/core/log4j-jul-%latest%.jar                    always
 lib/core/log4j-slf4j-impl-%latest%.jar             always
 lib/core/slf4j-api-%latest%.jar                    always
-lib/core/pkg-repo.jar                              always
+lib/core/pkg-java-fork.jar                         always
```

Comment:
Nope, my switch to Java NIO is not complete yet. More work needs to be done, but for now, the goal is just to get rid of those issues plaguing windows users in particular.